_partials/_timescaledb.md

```diff
@@ -0,0 +1,5 @@
+TimescaleDB is an extension for PostgreSQL that enables time-series workloads, 
+increasing ingest, query, storage and analytics performance. 
+We recommend running TimescaleDB on Timescale in the cloud, 
```

Comment:
Best practice is to run Timescale ... .  If you want to self host, you easily add Timescale functionality to your PostgreSQL instance.